src/metpy/calc/tools.py

```diff
@@ -1246,6 +1489,202 @@ def laplacian(f, axes=None, coordinates=None, deltas=None):
     return sum(derivs)
 
 
+@exporter.export
+@parse_grid_arguments
+@preprocess_and_wrap(wrap_like=('u', 'u', 'u', 'u'),
```

Comment:
This _should_ work given the `zip`-based implementation in `preprocess_and_wrap`, but given that implementation-dependence, it would be nice to see a test with this using a subset of the `return_only` options.